heal/src/glfs-heal.c

```diff
@@ -1000,8 +1002,11 @@ glfsh_print_pending_heals(glfs_t *fs, xlator_t *top_subvol, loc_t *rootloc,
         ret = 0;
         goto out;
     }
+    if (glfsh_output)
+        ret = glfsh_output->print_brick_from_xl(xl, rootloc);
+    else
+        ret = -EINVAL;
```

Comment:
`EFAULT` would be more appropriate here as we are facing this error due to `glfsh_output` being NULL (bad memory).

Other than that, LGTM.

Regards